src/test/scala/com/actian/spark_vector/vector/VectorOpsTest.scala

```diff
@@ -350,13 +350,27 @@ class VectorOpsTest extends fixture.FunSuite with SparkContextFixture with Match
 
   private def compareResults(result: Seq[Seq[Any]], expectedData: Seq[Seq[Any]], mappedIndices: Set[Int]) = {
     result.size should be(expectedData.size)
-    Inspectors.forAll(result.sortBy(_.mkString).zip(expectedData.sortBy(_.mkString))) {
-      case (actRow, expRow) =>
-        actRow.size should be(expRow.size)
-        Inspectors.forAll(actRow.zip(expRow).zipWithIndex.filter(i => mappedIndices.contains(i._2)).map(_._1)) {
-          case (act: Date, exp: Date) => Math.abs(act.getTime - exp.getTime) / MillisecondsInDay should be(0)
-          case (act, exp) => act should be(exp)
-        }
+    if (result.size > 0) {
+      Inspectors.forAll(result) {
+        case actRow =>
+          expectedData.find {
+            case expRow =>
+              expRow.size == actRow.size &&
+                {
```

Comment:
nit: move the `{` on the line before, or delete it since I don't think it's super usefull